app/views/application_records/show.haml

```diff
@@ -6,6 +6,10 @@ by:
 = @record.user.name_and_email
 = button_to 'Print this page',
   application_record_path(@record, format: :pdf), method: :get
+- if !@record.saved_for_later && @record.note_for_later.present?
```

Comment:
@Anbranin and I talked and decided that requiring a note is the simplest way to resolve this.